tasks/config.js

```diff
@@ -27,6 +28,7 @@ module.exports = {
     default: 'dist',
     templates: 'dist',
     www_pages: 'dist',
+    www_code_pages: 'dist/www/code',
```

Comment:
are we pushing it in the same file? we should put it in www/hl/<same-file-name> ?